README.mkd

```diff
@@ -279,7 +281,7 @@ Vendors should also aim to follow the conventions used for naming mnemonics
 in the ratified base ISA and extensions (e.g. the use of 'w', 'd',
 'u', and 's' suffixes).
 
-The vendor suffix requires at least two letters.
+The vendor prefix requires at least two letters.
```

Comment:
I think this should be moved a paragraph up, to be next to the other paragraphs about the prefixes. Also, keeping with our "must" -> "should" replacements, this should probably say something like "The vendor prefix should be at least two letters long".

(BTW, I wonder if we should say "characters" instead of "letters"?)

Reply:
> I think this should be moved a paragraph up, to be next to the other paragraphs about the prefixes. Also, keeping with our "must" -> "should" replacements, this should probably say something like "The vendor prefix should be at least two letters long".

Moved, that make sense to me .

> (BTW, I wonder if we should say "characters" instead of "letters"?)

Thanks for the suggestion, I don't have strong feeling on the difference between characters and letters, so I ask google translation and it give me `two letters` :P